src/sentry/static/sentry/app/components/version.tsx

```diff
@@ -125,12 +127,26 @@ const Version = ({
     </TooltipContent>
   );
 
+  const getPopperStyles = () => {
+    // if the version name is not a hash and we are not on mobile, allow tooltip to be as wide as 500px
+    if (/^[a-f0-9]{40}$/.test(version)) {
```

Comment:
Newer git can have 64 byte hashes as they will be moving to sha265 instead of sha1.

Reply:
Good point!